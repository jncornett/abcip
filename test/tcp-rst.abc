a ( syn )
b ( syn, ack )
a ( ack )

a ( len=16 )

b ( rst )

a ( len=16 )

