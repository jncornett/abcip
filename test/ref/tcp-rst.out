IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto TCP (6), length 40)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [S], cksum 0xb3d9 (correct), seq 1, win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0028 0001 0000 4006 5cbc 0a01 0203 0a09  .(....@.\.......
	0x0020:  0807 bdec 0008 0000 0001 0000 0000 5002  ..............P.
	0x0030:  2000 b3d9 0000                           ......
IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto TCP (6), length 40)
    10.9.8.7.8 > 10.1.2.3.48620: Flags [S.], cksum 0xb3c7 (correct), seq 1, ack 2, win 8192, length 0
	0x0000:  0201 0203 0405 0209 0807 0605 0800 4500  ..............E.
	0x0010:  0028 0001 0000 4006 5cbc 0a09 0807 0a01  .(....@.\.......
	0x0020:  0203 0008 bdec 0000 0001 0000 0002 5012  ..............P.
	0x0030:  2000 b3c7 0000                           ......
IP (tos 0x0, ttl 64, id 2, offset 0, flags [none], proto TCP (6), length 40)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [.], cksum 0xb3c8 (correct), ack 1, win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0028 0002 0000 4006 5cbb 0a01 0203 0a09  .(....@.\.......
	0x0020:  0807 bdec 0008 0000 0002 0000 0002 5010  ..............P.
	0x0030:  2000 b3c8 0000                           ......
IP (tos 0x0, ttl 64, id 3, offset 0, flags [none], proto TCP (6), length 56)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [none], cksum 0x6cf0 (correct), seq 2:18, win 8192, length 16
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0038 0003 0000 4006 5caa 0a01 0203 0a09  .8....@.\.......
	0x0020:  0807 bdec 0008 0000 0002 0000 0000 5000  ..............P.
	0x0030:  2000 6cf0 0000 4c6f 7265 6d20 6970 7375  ..l...Lorem.ipsu
	0x0040:  6d20 646f 6c6f                           m.dolo
IP (tos 0x0, ttl 64, id 2, offset 0, flags [none], proto TCP (6), length 40)
    10.9.8.7.8 > 10.1.2.3.48620: Flags [R], cksum 0xb3d6 (correct), seq 2, win 8192, length 0
	0x0000:  0201 0203 0405 0209 0807 0605 0800 4500  ..............E.
	0x0010:  0028 0002 0000 4006 5cbb 0a09 0807 0a01  .(....@.\.......
	0x0020:  0203 0008 bdec 0000 0002 0000 0000 5004  ..............P.
	0x0030:  2000 b3d6 0000                           ......
IP (tos 0x0, ttl 64, id 4, offset 0, flags [none], proto TCP (6), length 56)
    10.1.2.3.48620 > 10.9.8.7.8: Flags [none], cksum 0x952b (correct), seq 18:34, win 8192, length 16
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0038 0004 0000 4006 5ca9 0a01 0203 0a09  .8....@.\.......
	0x0020:  0807 bdec 0008 0000 0012 0000 0000 5000  ..............P.
	0x0030:  2000 952b 0000 7220 7369 7420 616d 6574  ...+..r.sit.amet
	0x0040:  2c20 636f 6e73                           ,.cons
